src/tiler/server.js

```diff
@@ -56,12 +64,41 @@ var config = {
     },
     log_format: '{ "timestamp": ":date[iso]", "@fields": { "remote_addr": ":remote-addr", "body_bytes_sent": ":res[content-length]", "request_time": ":response-time", "status": ":status", "request": ":method :url HTTP/:http-version", "request_method": ":method", "http_referrer": ":referrer", "http_user_agent": ":user-agent" } }',
 
+    enable_cors: true,
+
     beforeTileRender: function(req, res, callback) {
         callback(null);
     },
 
     afterTileRender: function(req, res, tile, headers, callback) {
+
+        // Complete render pipline first
+        headers['Cache-Control'] = 'max-age=2592000';
```

Comment:
Happens to be 30 days, but is significant only in that it is a long time.  Taken from otm/nyc-trees.